src/Picker/ArticlePickerProvider.php

```diff
@@ -51,10 +55,29 @@ public function getDcaTable(): string
      */
     public function getDcaAttributes(PickerConfig $config): array
     {
+        $value = $config->getValue();
         $attributes = ['fieldType' => 'radio'];
 
-        if ($this->supportsValue($config)) {
-            $attributes['value'] = str_replace(['{{article_url::', '}}'], '', $config->getValue());
+        if ('article' === $config->getContext()) {
+            $attributes = ['fieldType' => $config->getExtra('fieldType')];
```

Comment:
remove this line, it overrides the default in line 59 and is duplicate with line 65